src/components/MessageInput/MessageInput.tsx

```diff
@@ -332,7 +334,17 @@ export const MessageInputWithContext = <
       inputBoxRef.current.focus();
     }
 
-    if (!editing) {
+    /**
+     * Make sure to test `initialValue` functionality, if you are modifying following condition.
```

Comment:
potentially add in what you think would be good to test here? as it's fresh in your mind but may require added thought in the future and would simplify life then if we knew exactly what to check